frontend/common/code-help/traits/traits-dotnet.js

```diff
@@ -1,12 +1,20 @@
-module.exports = (envId, { LIB_NAME, USER_ID, TRAIT_NAME, LIB_NAME_JAVA, FEATURE_NAME, FEATURE_FUNCTION, FEATURE_NAME_ALT, FEATURE_NAME_ALT_VALUE, NPM_CLIENT }, userId) => `
-// Set a user trait
-Trait userTrait = await BulletTrainClient.instance.SetTrait("${USER_ID}", "${TRAIT_NAME}", "blue");
+module.exports = (envId, { LIB_NAME, USER_ID, TRAIT_NAME, LIB_NAME_JAVA, FEATURE_NAME, FEATURE_FUNCTION, FEATURE_NAME_ALT, FEATURE_NAME_ALT_VALUE, NPM_CLIENT }, userId) => `using Flagsmith;
 
-// Increment a trait
-Trait userTrait = await BulletTrainClient.instance.IncrementTrait("${USER_ID}", "${TRAIT_NAME}", 1);
+static FlagsmithClient _flagsmithClient;
+_flagsmithClient = new("${envId}");
 
-Identity userIdentity = await BulletTrainClient.instance.GetUserIdentity("${USER_ID}");
-if (userIdentity != null) {
-  // Run the code to use user identity i.e. userIdentity.flags or userIdentity.traits
-}
+var Identifier = "delboy@trotterstraders.co.uk";
+var traitKey = "car_type";
+var traitValue = "robin_reliant";
+var traitList = new List<Trait> { new Trait(traitKey, traitValue) };
+
+# Sync
```

Comment:
I don't think this is right, is it? Both of these will just return a `Task` and probably the call to `isFeatureEnabled` will actually fail in fact. 

I think you need to add `.Result` to the end of each of these?